core/templates/dev/head/css/oppia.css

```diff
@@ -177,6 +172,10 @@ md-input-group.md-default-theme label {
   text-decoration: none;
 }
 
+.btn.btn-secondary.oppia-revert-button {
```

Comment:
Should this be placed with the other .btn classes (below)? (I'm not sure; just checking.)

Reply:
i kept it there because it is specific to oppia-revert-button i have added btn classes in the front to override precedence Thanks!